app/src/main/java/protect/card_locker/preferences/SettingsActivity.java

```diff
@@ -148,10 +158,20 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
                 refreshActivity(true);
                 return true;
             });
+            colorPreference.setSummary(colorPreference.getEntry());
```

Comment:
This seems to be slightly buggy on Android pre-Material You Android versions (Android 11 and before, 12 and before for non-Pixels). If I choose the "Catima" theme, the text below "Theme color" says "System" instead.

Reply:
Oh, I tested this on Android 13 and worked fine for me, will look for fixes